health/health_config.c

```diff
@@ -652,25 +717,30 @@ static int health_readfile(const char *filename, void *data) {
             rc->old_status = RRDCALC_STATUS_UNINITIALIZED;
             rc->warn_repeat_every = host->health_default_warn_repeat_every;
             rc->crit_repeat_every = host->health_default_crit_repeat_every;
+            alert_cfg = callocz(1, sizeof(struct alert_config));
```

Comment:
memory leak if alert_cfg was allocated already

Reply:
Ok, added free of `cfg` inside `alert_config_free`